vehicle/common/charging/charging.yml

```diff
@@ -18,34 +18,34 @@ publish:
           values:
             t: Charger connected
             f: Charger disconnected
-name: elcon_hk_j_440_10
```

Comment:
we should figure out how to work with the can_api to use this common message rather than needing to copy-paste it - we can work on this together when we're back or you can poke around and try to see what's up